mantle/platform/qemu.go

```diff
@@ -775,6 +775,15 @@ func (builder *QemuBuilder) AddIso(path string, bootindexStr string) error {
 	return nil
 }
 
+// AddQMP enables QMP used to interact with the qemu VM
+func (builder *QemuBuilder) AddQMP(tempdir string) error {
```

Comment:
I think we could just do this unconditionally inside `Exec()` too.

Reply:
yes done. that cleans up a bit too